protocols/v1/src/methods/client_to_server.rs

```diff
@@ -18,6 +18,53 @@ use quickcheck::{Arbitrary, Gen};
 #[cfg(test)]
 use quickcheck_macros;
 
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
I dont think we need the `Clone` or the `PartialEq` here

Reply:
I think we should include clone, the other msg types implement clone as well.